cardano-cli/src/Cardano/CLI/Shelley/Run/Transaction.hs

```diff
@@ -113,6 +113,8 @@ data ShelleyTxCmdError
   | ShelleyTxCmdTxInsDoNotExist ![TxIn]
   | ShelleyTxCmdMinimumUTxOErr !MinimumUTxOError
   | ShelleyTxCmdPParamsErr !ProtocolParametersError
+  | ShelleyTxCmdUnsupportedVersion !MinNodeToClientVersion !NodeToClientVersion
```

Comment:
So this should be `ShelleyTxCmdQueryError QueryError`

Reply:
I  merged `ShelleyQueryCmdAcquireFailure` and `ShelleyQueryCmdUnsupportedVersion` constructors into `ShelleyQueryCmdQueryError`, I think this was what you meant?